### ISSUE TITLE
Blocks three finger drag

### ISSUE THREAD

**Issue by psyf4ctor** [artginzburg/MiddleClick-BigSur#48]
Is there any possibility to make them both work?

**Comment by artginzburg** [comment]
I suppose it's a known issue that should either be fixed or added to Readme. Do you have any additional problems, like #34? Don't have much time to research this now, but we can at least organise the issues meanwhile.

**Comment by moranno** [comment]
support 4 finger click/tab to solve this issue.

**Comment by artginzburg** [comment]
> support 4 finger click/tab to solve this issue.

It's supported, take a look here: https://github.com/artginzburg/MiddleClick-Ventura#number-of-fingers
Perhaps the app should have a preferences window with a slider range for the number of fingers. But for now, I don't see anyone actually asking for this, so it can be left out.

**Comment by SuperLeiyan** [comment]
Is this still issue still being looked at? I badly need the three finger drag to work with this

**Comment by danieltsoftware** [comment]
> Is this still issue still being looked at? I badly need the three finger drag to work with this

You can hold down the option key to makr dragging work. It's a MacOS thing, I checked the code and it cannot be changed to be default behavior. If you need it for Unity I have made a script to allow for middle mouse dragging using this app without the need to constantly hold the option key. [Unity Magic Mouse Panning on GitHub](https://github.com/danieltsoftware/UnityPanningMagicMouse)

**Comment by SuperLeiyan** [comment]
BetterTouchTool's three finger click doesn't seem to interfere, don't really want to pay for it since I just need this one functionality, I wonder how it's implemented in BTT.

**Comment by SuperLeiyan** [comment]
> > Is this still issue still being looked at? I badly need the three finger drag to work with this
> 
> You can hold down the option key to make dragging work. It's a MacOS thing, I checked the code and it cannot be changed to be default behavior. If you need it for Unity I have made a script to allow for middle mouse dragging using this app without the need to constantly hold the option key. [Unity Magic Mouse Panning on GitHub](https://github.com/danieltsoftware/UnityPanningMagicMouse)

BetterTouchTool's three finger click doesn't seem to interfere, don't really want to pay for it since I just need this one functionality, I wonder how it's implemented in BTT.

**Comment by danieltsoftware** [comment]
> > > Is this still issue still being looked at? I badly need the three finger drag to work with this
> > 
> > 
> > You can hold down the option key to make dragging work. It's a MacOS thing, I checked the code and it cannot be changed to be default behavior. If you need it for Unity I have made a script to allow for middle mouse dragging using this app without the need to constantly hold the option key. [Unity Magic Mouse Panning on GitHub](https://github.com/danieltsoftware/UnityPanningMagicMouse)
> 
> BetterTouchTool's three finger click and it doesn't seem to interfere, don't really want to pay for it since I just need this one functionality, I wonder how it's implemented in BTT.

BTT is absolutely terrible in my experience, constantly freezing up my inputs on my barnd new M3 Pro MacBook, and the middle mouse functionality barely works, on top of needing to be added through weird "custom mouse commands". I bought a Magic Mouse because I saw there is BTT, then when I got the mouse and downloaded BTT I promptly deleted it after an hour of plain horror and started looking for alternatives lol.

**Comment by danieltsoftware** [comment]
> > > Is this still issue still being looked at? I badly need the three finger drag to work with this
> > 
> > 
> > You can hold down the option key to make dragging work. It's a MacOS thing, I checked the code and it cannot be changed to be default behavior. If you need it for Unity I have made a script to allow for middle mouse dragging using this app without the need to constantly hold the option key. [Unity Magic Mouse Panning on GitHub](https://github.com/danieltsoftware/UnityPanningMagicMouse)
> 
> BetterTouchTool's three finger click and it doesn't seem to interfere, don't really want to pay for it since I just need this one functionality, I wonder how it's implemented in BTT.

Also, did you try [middleclick.app](https://middleclick.app/)?

**Comment by SuperLeiyan** [comment]
> > > > Is this still issue still being looked at? I badly need the three finger drag to work with this
> > > 
> > > 
> > > You can hold down the option key to make dragging work. It's a MacOS thing, I checked the code and it cannot be changed to be default behavior. If you need it for Unity I have made a script to allow for middle mouse dragging using this app without the need to constantly hold the option key. [Unity Magic Mouse Panning on GitHub](https://github.com/danieltsoftware/UnityPanningMagicMouse)
> > 
> > 
> > BetterTouchTool's three finger click and it doesn't seem to interfere, don't really want to pay for it since I just need this one functionality, I wonder how it's implemented in BTT.
> 
> Also, did you try [middleclick.app](https://middleclick.app/)? It's from the same dev as Rectangle

Yeah I've tried it, also blocks the three finger drag, so far only BTT works.

**Comment by artginzburg** [comment]
This commit is worth having a look 004510c78801852ffdb3de4ed4527db2ebf1d010. Apparently, not all commits of the previous contributors were merged
